charts/gitops-server/values.yaml

```diff
@@ -119,6 +119,8 @@ ingress:
   #  - secretName: chart-example-tls
   #    hosts:
   #      - chart-example.local
+extraVolumes: []
+extraVolumeMounts: []
```

Comment:
I believe it'd be convenient for users to provide examples for both of these settings to understand the syntax.